benchmarks/utils/report.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+
+METRICS_VALIDATED = [
+    "throughput",
+    "total_latency_p50",
+    "model_latency_p50",
+    "total_latency_p90",
+    "model_latency_p90",
+    "total_latency_p99",
+    "model_latency_p99",
+    "memory_percentage_mean",
+    "gpu_used_memory_mean",
+    "cpu_percentage_mean",
+    "gpu_percentage_mean",
+]
+
+
+# Acceptable metric deviation needs a more complicated logic.
+# Example: For latencies in 2 digits, 50% might be acceptable
+# For 3 digit latencies, 20-30% might be the right value
+# For cpu_memory < 15%, 50% deviation works but for CPU > 40%, 10-15%
+# might be the right value
+ACCEPTABLE_METRIC_DEVIATION = 0.3
+
+
+class Report:
+    def __init__(self):
+        self.properties = {}
+        self.mode = None
+        self.throughput = 0
+        self.batch_size = 0
+        self.workers = 0
+        self.deviation = ACCEPTABLE_METRIC_DEVIATION
```

Comment:
pass the deviation instead to init with default set to 0.3

Reply:
Great idea. Done